Automattic/newspack-theme
# feat: add Portuguese translations to the theme

### All Submissions:

* [x ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds a Portuguese translation to the theme.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to WP Admin > Settings and change your language to Portuguese. 
3. View the site on the front-end; confirm that strings like bylines, and the Newspack credit in the footer are translated. 
4. Edit a post; confirm that some strings in the editor under "Imagem de destaque" ("Featured Image") are translated - the focus of this set is the front end, but it should say "Padrão" instead of "Default", "Amplo" instead of "Large"... this is to confirm that the editor is picking up the JSON files. 

### Other information:
